BNomad/View/MapView/MapViewController.swift

```diff
@@ -224,24 +218,38 @@ class MapViewController: UIViewController {
         
     }
     
-    func setStartingRegion() {
-        guard let user = viewModel.user else { return }
-        if user.isChecked {
-            print("체크인 상태로 맵 세팅")
-//            guard let place = checkedPlace else { return }
-//            map.setRegion(MKCoordinateRegion(center: CLLocationCoordinate2D(latitude: place.latitude, longitude: place.longitude), span: MKCoordinateSpan(latitudeDelta: 0.01, longitudeDelta: 0.01)), animated: true)
-            
-            guard let location = currentLocation else { return }
-            map.setRegion(MKCoordinateRegion(center: location.coordinate, span: MKCoordinateSpan(latitudeDelta: 0.1, longitudeDelta: 0.1)), animated: true)
-
-            
-            print("체크인 상태로 맵 세팅")
-        } else {
-            print("체크아웃 상태로 맵 세팅")
-            guard let location = currentLocation else { return }
-            
-            map.setRegion(MKCoordinateRegion(center: location.coordinate, span: MKCoordinateSpan(latitudeDelta: 0.1, longitudeDelta: 0.1)), animated: true)
-            print("체크아웃 상태로 맵 세팅")
+    func setMapRegion(_ latitude: Double, _ longitude: Double, spanDelta: Double) {
+        map.setRegion(MKCoordinateRegion(center: CLLocationCoordinate2D(latitude: latitude, longitude: longitude), span: MKCoordinateSpan(latitudeDelta: spanDelta, longitudeDelta: spanDelta)), animated: true)
+    }
+    
+    func checkInBinding() {
+        print("체크인 바인딩 -> 위치 전달")
+        if let user = viewModel.user {
+            if user.isChecked {
```

Comment:
[P3] 아래로 수정하면 중첩 하나를 제거할 수 있을 것 같습니다!
```suggestion
       if let isCheckd = viewModel.user?.isChecked {
```

Reply:
user가 옵셔널 값이라서 한 번 unwrap 해준 뒤에 체크인 Bool만 가지고 if 분기를 하려고 했습니다.
추천해주신대로 한 줄로 시도를 해봤는데 초기화면 로드가 제대로 안되네요 ㅠ
한 줄로 하면 user의 존재 여부 optional과 Bool 이 한 데 합쳐져서 제대로 분리가 안되는 것 같습니다.